src/main/java/io/orten/nano/util/Database.java

```diff
@@ -1,64 +1,113 @@
+                .configure("hibernate.cfg.xml") // configures settings from hibernate.cfg.xml
+                .build();
+        MetadataSources mds = new MetadataSources(registry);
+        Metadata md = mds.buildMetadata();
+        sessionFactory = md.buildSessionFactory();
+        //Make sure that the service registry is destroyed on shutdown by adding a shutdown hook to the runtime
+        Runtime.getRuntime().addShutdownHook(new Thread(() ->
+        {
+            StandardServiceRegistryBuilder.destroy(registry);
+        }));
+    }
+
+    //saves an organization's object to the database
+
+    public static void saveOrganization(Organization org) {
+        if (sessionFactory == null)
+            init();
+        try (Session s = sessionFactory.openSession()) {
+            s.beginTransaction();
+            s.save(org);
+            s.getTransaction().commit();
         }
+    }
+  
+    //updates an organization's object already saved in saved in the database
 
-        /**
-         * saving an organization object to the database
-        */
-        public static void saveObject(Object o){
-            if(sessionFactory == null)
-                init();
-            try(Session s = sessionFactory.openSession()){
-                s.beginTransaction();
-                s.save(o);
-                s.getTransaction().commit();
-            }
+    public static void updateOrganization(Organization org) {
+        if (sessionFactory == null)
+            init();
+        try (Session s = sessionFactory.openSession()) {
+            s.beginTransaction();
+            s.update(org);
+            s.getTransaction().commit();
         }
 
-        /**
-         * updating an organization object
-        */
+    }
+
+    //gets one organization's object from the database based on its ID
 
-        public static void updateOrganization(Object o){
-            if(sessionFactory == null)
-                init();
-            try(Session s = sessionFactory.openSession()){
-                s.beginTransaction();
-                s.update(o);
-                s.getTransaction().commit();
-            }
+    public static List<Organization> getOrganization(String orgID) {
+        if (sessionFactory == null)
+            init();
+        try (Session s = sessionFactory.openSession()) {
+            s.beginTransaction();
+            Query q = s.createQuery("from Organization as org where org.organizationID = :orgID");
+            q.setParameter("orgID", orgID);
+            List<Organization> list = q.getResultList();
+            s.getTransaction().commit();
+            return list;
+        }      
+     }
+      
+    //gets all the organization objects from the database
+
+    public static List<Organization> getAllOrganizations(){
+        if (sessionFactory == null)
+            init();
+        try (Session s = sessionFactory.openSession()) {
+            s.beginTransaction();
```

Comment:
You don't need to begin(or commit) a transaction if you are just retrieving data